gamification/src/main/java/com/appcoins/wallet/gamification/repository/entity/VipReferralResponse.kt

```diff
@@ -0,0 +1,27 @@
+package com.appcoins.wallet.gamification.repository.entity
+
+import com.google.gson.annotations.SerializedName
+
+data class VipReferralResponse(
+  @SerializedName("code") val code: String?,
```

Comment:
Usually I prefer this way as a safety measure, in case the backend returns only some of the fields, so that we can then substitute with default values. Changed the "code" field to non-nullable anyway.